pennylane/transforms/control.py

```diff
@@ -101,6 +100,10 @@ def __init__(self, tape, control_wires, do_queue=True):
         wires = self.control_wires + tape.wires
         super().__init__(*tape.get_parameters(), wires=wires, do_queue=do_queue)
 
+    @property
+    def num_params(self):
+        return property(lambda self: self.subtape.num_params)

```

Comment:
This seems to have caused a lot of tests to fail in `transforms/test_control.py`. Might be because we're still trying to access `num_params` through the `subtape`?

Reply:
I'm an idiot - tired to do this in a minute. You're right, it should just return `self.subtape.num_params`. Sorry :)